modules/runner/src/integrationTest/java/org/apache/ignite/internal/inmemory/ItRaftStorageVolatilityTest.java

```diff
@@ -163,11 +162,12 @@ void raftMetaStorageIsPersistentForPersistentPartitions() {
     }
 
     private void createPersistentTable() {
-        executeSql("CREATE ZONE ZONE_" + TABLE_NAME + " ENGINE rocksdb WITH STORAGE_PROFILES = '" + DUMMY_STORAGE_PROFILE + "'");
+        executeSql("CREATE ZONE ZONE_" + TABLE_NAME
+                + " ENGINE rocksdb WITH DATAREGION = 'default_rocksdb', STORAGE_PROFILES = 'default_rocksdb'");
```

Comment:
why do we need to set here DATAREGION?

Reply:
It is still using in the Catalog, I will remove it under https://issues.apache.org/jira/browse/IGNITE-21385

Reply:
I do not understand why your changes leads to setting this filed?

Reply:
Because earlier it was working because of setting dataRegion to "default" on the parsing sql query. But now storage engines has no "default" regions, so, we need to set it manually here.

Reply:
I rechecked all tests with custom storage_profiles. Actually we set it in all tests, which use not "default" storage profiles. In all other tests, we use this hint now.

Just want to note, in the next PR for https://issues.apache.org/jira/browse/IGNITE-21385 I will remove this duality at all, together with the dataregion field from zone ddl